documentation-templates/README.md

```diff
@@ -0,0 +1,19 @@
+# documentation-templates
+
+The templates available in [scaffolder-templates](https://github.com/parodos-dev/workflow-software-templates/tree/main/scaffolder-templates) are integrated in the Backstage Orchestrator plugin under `locations` in the following deployment manifest [values](https://github.com/anludke/orchestrator-helm-chart/blob/main/charts/orchestrator/values.yaml#L121).
+
+## basic-workflow
+This template allows the user to generate a basic workflow project from the user's inputs collected.
+Based on the workflow type (assessment, infrastructure) the user selects, the corresponding workflow template is generated.
+
+### basic-workflow as assessment
```

Comment:
pls mention the expected output schema format of the assessment workflow which differentiates it from the infrastructure workflow and enables it to be shown in the UI.

Reply:
Done.